include/extractor/guidance/sliproad_handler.hpp

```diff
@@ -42,6 +47,40 @@ class SliproadHandler : public IntersectionHandler
     Intersection operator()(const NodeID nid,
                             const EdgeID via_eid,
                             Intersection intersection) const override final;
+
+  private:
+    util::ScopedGeojsonLoggerGuard<util::NodeIdVectorToLineString> geojson_lines;
+    util::ScopedGeojsonLoggerGuard<util::NodeIdVectorToMultiPoint> geojson_points;
+
+    boost::optional<std::size_t> getObviousIndexWithSliproads(const EdgeID from,
+                                                              const Intersection &intersection,
+                                                              const NodeID at) const;
+
+    // Next intersection from `start` onto `onto` is too far away for a Siproad scenario
+    bool nextIntersectionIsTooFarAway(const NodeID start, const EdgeID onto) const;
+
+    // Through street: does a road continue with from's name at the intersection
+    bool isThroughStreet(const EdgeID from, const IntersectionView &intersection) const;
```

Comment:
can you add a ticket to the guidance refactor project for removing this duplication with the intersection handler?